### DISCUSSION TITLE
6772691992466f8c6ec39e43

### DISCUSSION THREAD

**Discussion by giscus** [Admin-ToxicGames/toxicgames#13] [General]
# 6772691992466f8c6ec39e43

Discover a wide range of cracked games, mods, and gaming resources for PC, Mac, and more. Download your favorite games for free and enjoy endless gaming adventures!

https://toxicgames.in/6772691992466f8c6ec39e43

**Comment by Admin-ToxicGames** [comment]
<h2>Change Language</h2>


```markdown
> The language changes in Launcher, you can set English voice acting and Russian subtitles by checking the corresponding checkbox or set Russian voice acting and Russian subtitles. 

Language is set in Launcher you need to put both subtitles and the language of the audio.

>Useful information that users should know, even when skimming content.

**Comment by agarwaldivyanshh** [comment]
my cursor is getting stuck in a virtual box 
and i am not able to proceed further because of that 
the graphics are also limited to 1440 *900 not even 1920*1080

tell me the fix pls
